app/content/js/views/list/activations.js

```diff
@@ -36,7 +36,7 @@ const fetch = activationIds => Promise.all(activationIds.map(_ => {
  *
  */
 const show = activation => () => {
-    if (activation.logs && activation.logs.length === 1) {
+    if (activation.logs && activation.logs.length === 1 && activation.logs[0].length === 32) {
```

Comment:
what does the 32 mean?

Reply:
32 is the length of an activation/session id 

the original if-statement assumes that if an activation has logs length == 1 then the log is a session id, which is not true. i added logs[0].length == 32 as an additional requirement

Reply:
did that change recently?

Reply:
and, in any case, at least a comment. and perhaps a regex check, instead?